lib/pbench/server/api/resources/server_configuration.py

```diff
@@ -200,7 +210,9 @@ def _put_body(self, params: ApiParams) -> Response:
             raise APIAbort(fail_status, message=", ".join(failures))
         return jsonify(response)
 
-    def _put(self, params: ApiParams, _) -> Response:
+    def _put(
+        self, params: ApiParams, request: Request, context: ApiContext
+    ) -> Response:
```

Comment:
The docstring "Args" section doesn't mention `context`.

Reply:
Dang; I thought I added it, but I'm not really surprised that I missed some.